apps/web/src/components/dashboard/energy-consumption-statistics.tsx

```diff
@@ -58,38 +58,22 @@ export default async function EnergyConsumptionStatisticCard({ startDate, endDat
             <CardHeader>
                 <CardTitle>Verbrauchsstatistiken</CardTitle>
                 <CardDescription>
-                    {startDate.toDateString() === endDate.toDateString() ? (
-                        <>
-                            {format(startDate, "PPP", {
-                                locale: de,
-                            })}
-                        </>
-                    ) : (
-                        <>
-                            {format(startDate, "PPP", {
-                                locale: de,
-                            })}{" "}
-                            -{" "}
-                            {format(endDate, "PPP", {
-                                locale: de,
-                            })}
-                        </>
-                    )}
+                    {format(startDate, "PPP", {locale: de})} - {format(endDate, "PPP", {locale: de})}
                 </CardDescription>
             </CardHeader>
             <CardContent>
                 <div className="grid grid-cols-3 gap-4">
                     <div>
                         <h2 className="text-center text-xl font-semibold text-primary">Max.</h2>
-                        <p className="text-center">{maxConsumption} kWh</p>
+                        <p className="text-center">{maxConsumption.toLocaleString('de-DE', {minimumFractionDigits: 2, maximumFractionDigits: 2})} kWh</p>
```

Comment:
Kannst du das Formatieren bitte in eine eigene Funktion auslagern? Du verwendest das ja jetzt an mehreren Stellen. Dann bräuchte man z. B. nur einen simplen Methodenaufruf machen:
`formatNumber(maxConsumption)`